### PR TITLE
CO-3174 Server action ids don't exists anymore in base.automation

### PR DESCRIPTION
**Pull request by quentingigon**
I am not sure that it is the correct fix

### CODE REVIEW

File: intervention_compassion/data/intervention_action_rules.xml [hunk 1]
```diff
@@ -23,11 +23,19 @@
...
 
     <!-- Rule for checking SLA Negociation -->
-    <record id="intervention_sla_check" model="base.automation">
+    <record id="intervention_sla_check_done" model="base.automation">
         <field name="name">Intervention: Check SLA Negociation</field>
         <field name="model_id" ref="model_compassion_intervention"/>
         <field name="trigger">on_create_or_write</field>
         <field name="filter_domain">[('state', 'in', ['on_hold', 'sla'])]</field>
-        <field name="server_action_ids" eval="[(6, 0, [ref('intervention_sla_done_action'), ref('intervention_sla_wait_action')])]"/>
+        <field name="server_action_id" eval="[(6, 0, [ref('intervention_sla_done_action')])]"/>
```
Comment (ecino) [inline intervention_compassion/data/intervention_action_rules.xml]:
```suggestion
        <field name="action_server_id" ref="intervention_sla_done_action"/>
```

File: intervention_compassion/data/intervention_action_rules.xml [hunk 2]
```diff
@@ -23,11 +23,19 @@
...
+    </record>
+
+    <record id="intervention_sla_check_wait" model="base.automation">
+        <field name="name">Intervention: Check SLA Negociation</field>
+        <field name="model_id" ref="model_compassion_intervention"/>
+        <field name="trigger">on_create_or_write</field>
+        <field name="filter_domain">[('state', 'in', ['on_hold', 'sla'])]</field>
+        <field name="server_action_id" eval="[(6, 0, [ref('intervention_sla_wait_action')])]"/>
```
Comment (ecino) [inline intervention_compassion/data/intervention_action_rules.xml]:
```suggestion
        <field name="action_server_id" ref="intervention_sla_wait_action"/>
```

File: intervention_compassion/data/intervention_action_rules.xml [hunk 3]
```diff
@@ -23,11 +23,19 @@
     </record>
 
     <!-- Rule for checking SLA Negociation -->
-    <record id="intervention_sla_check" model="base.automation">
+    <record id="intervention_sla_check_done" model="base.automation">
         <field name="name">Intervention: Check SLA Negociation</field>
```
Comment (ecino) [inline intervention_compassion/data/intervention_action_rules.xml]:
```suggestion
        <field name="name">Intervention: Check SLA Negociation done</field>
```

File: intervention_compassion/data/intervention_action_rules.xml [hunk 4]
```diff
@@ -23,11 +23,19 @@
...
         <field name="model_id" ref="model_compassion_intervention"/>
         <field name="trigger">on_create_or_write</field>
         <field name="filter_domain">[('state', 'in', ['on_hold', 'sla'])]</field>
-        <field name="server_action_ids" eval="[(6, 0, [ref('intervention_sla_done_action'), ref('intervention_sla_wait_action')])]"/>
+        <field name="server_action_id" eval="[(6, 0, [ref('intervention_sla_done_action')])]"/>
+    </record>
+
+    <record id="intervention_sla_check_wait" model="base.automation">
+        <field name="name">Intervention: Check SLA Negociation</field>
```
Comment (ecino) [inline intervention_compassion/data/intervention_action_rules.xml]:
```suggestion
        <field name="name">Intervention: Check SLA Negociation waiting</field>
```

**Review by ecino — CHANGES_REQUESTED** [review]

File: intervention_compassion/data/intervention_action_rules.xml [hunk 5]
```diff
@@ -23,11 +23,19 @@
...
     <!-- Rule for checking SLA Negociation -->
-    <record id="intervention_sla_check" model="base.automation">
-        <field name="name">Intervention: Check SLA Negociation</field>
+    <record id="intervention_sla_check_done" model="base.automation">
+        <field name="name">Intervention: Check SLA Negociation done</field>
         <field name="model_id" ref="model_compassion_intervention"/>
         <field name="trigger">on_create_or_write</field>
         <field name="filter_domain">[('state', 'in', ['on_hold', 'sla'])]</field>
-        <field name="server_action_ids" eval="[(6, 0, [ref('intervention_sla_done_action'), ref('intervention_sla_wait_action')])]"/>
+        <field name="server_action_id" ref="intervention_sla_done_action"/>
```
Comment (ecino) [inline intervention_compassion/data/intervention_action_rules.xml]: The field is named `action_server_id`
```suggestion
        <field name="action_server_id" ref="intervention_sla_done_action"/>
```

File: intervention_compassion/data/intervention_action_rules.xml [hunk 6]
```diff
@@ -23,11 +23,19 @@
...
+    </record>
+
+    <record id="intervention_sla_check_wait" model="base.automation">
+        <field name="name">Intervention: Check SLA Negociation waiting</field>
+        <field name="model_id" ref="model_compassion_intervention"/>
+        <field name="trigger">on_create_or_write</field>
+        <field name="filter_domain">[('state', 'in', ['on_hold', 'sla'])]</field>
+        <field name="server_action_id" ref="intervention_sla_wait_action"/>
```
Comment (ecino) [inline intervention_compassion/data/intervention_action_rules.xml]:
```suggestion
        <field name="action_server_id" ref="intervention_sla_wait_action"/>
```

**Review by ecino — CHANGES_REQUESTED** [review]

**Comment by quentingigon** [comment]
Sorry, I must not be super awake

**Review by ecino — APPROVED** [review]

**Pull request merged**